### ISSUE TITLE
Issue with Labels

### ISSUE THREAD

**Issue by Shekar73** [libreboard/libreboard#81]
Creating new labels and editing existing labels is failing

**Comment by Shekar73** [comment]
This feature seems to be working on my local branch with the latest commit, but not on www.libreboard.com

**Comment by mquandalle** [comment]
Hum this is strange. When you published this issue 4 hours ago I directly tried on my local instance and there was no bug, then I tried on the deployed instance and the bug was here. So I thought it was a problem of latency, or something like that.

But I just re-deployed the same commit again, and now it works in production. So maybe it was a mistake on my side (I deployed the wrong branch), but I don't think so because the deployed instance included the last commit I did, or maybe someone else (@yasaricli?) did deploy during the interval...

Anyway, it is "fixed" in both local and remote environments, so I'm closing this issue. Thank you for your will to help.
